ingestion-beam/src/test/java/com/mozilla/telemetry/decoder/ParsePayloadTest.java

```diff
@@ -322,4 +322,26 @@ public void testPingSplitting() throws IOException {
 
     pipeline.run();
   }
+
+  @Test
```

Comment:
Seems like we weren't testing this generally (and even specifically in some cases e.g. https://github.com/mozilla/gcp-ingestion/pull/2180) so you picked some arbitrary examples of stuff to verify dropping works :+1: